Extended Favicon Support

chridgely opened the pull request:
Since I got beat to the punch for the other issue, I just continued playing around with the icon and created files and support for other media. As well as moved the favicon to the root folder. This adds redundancy in case for whatever reason the icon does not get picked up by head, browsers will just automatically look for it in root. 

These are examples of what it would look like in other media forms...

![iphone](https://user-images.githubusercontent.com/9426990/31782509-1c4fd0e8-b4c9-11e7-976e-424cfbf5cd20.JPG)
![chrome phone](https://user-images.githubusercontent.com/9426990/31782519-21fae046-b4c9-11e7-86cd-15d6bb0eccfa.JPG)
![windows metro](https://user-images.githubusercontent.com/9426990/31782520-24920636-b4c9-11e7-8c32-a2d8a374cca3.JPG)
![safari](https://user-images.githubusercontent.com/9426990/31782522-25b6146c-b4c9-11e7-8bfd-f8ab99cfa1f3.JPG)

The app name will be picked up by the meta tag. I hope I'm not imposing with this pull request. It's not an issue but I thought it might be something you would be interested in and figured, why not try.

LukeSchlangen commented:
@chridgely This is awesome! The one issue I have with this is that everything is at the root level. Is it possible to move these files to a folder so that it keeps the root of the project clean?

chridgely commented:
Done! I kept favicon.ico in root because a lot of web browsers will expect it there. The remaining, I moved into images/icons and updated the references.

LukeSchlangen commented:
Perfect! Thank you so much!
